docs/business.yaml

```diff
@@ -4236,6 +4439,62 @@ components:
       description: The name of the legalName of the document.
       schema:
         type: string
+  responses:
```

Comment:
The responses code follows the PPR's pattern.

The examples provided are general scenarios. Not sure if we should expand the response examples to reflect specific types of filings. If so, this would involve detailing the possible error responses for different filing scenarios.